### ISSUE TITLE
Decrease capture delay while using finger photo gesture.

### ISSUE THREAD

**Issue by code807** [Yellow-Dog-Man/Resonite-Issues#867]
### Is your feature request related to a problem? Please describe.

While using a camera, I sometimes want to take photos somewhat rapidly, one after another. It is useful to have several shots from the same angle and roughly the same time to improve the chance of producing a high quality shot. The default finger photo gesture has a timeout after taking a photo, and makes me wait before I can take another. I feel as though the timeout is too long, and prevents me from taking photos fast enough.

### Describe the solution you'd like

I would like the timeout between taking being able to take finger photos to be decreased or removed.

### Describe alternatives you've considered

I could just deal with the timer and take less photos.

**Comment by ohzee00** [comment]
I don't think this is a intentional delay or some sort of timer, I think it is something wrong with the behavior itself that's taking longer

**Comment by Frooxius** [comment]
There's no delay, it's literally however fast you can make the gesture. I can take photos rapidly myself, like say at least 5 per second.

**Comment by ohzee00** [comment]
I'm confused on what you mean exactly @code807, do you mean to say the _gesture_ has a timeout or pictures are taking long to load in?
I believe I was mistaken when I first read what you said, because I know I can take rapid pictures with the gestures, they however take a fair bit of time (10-20 seconds) to load in.

**Comment by ExoTheWicker** [comment]
I've had a delay as much as 1 minute 30s after the shutter sound to when the picture loads in

**Comment by Frooxius** [comment]
For the pictures loading in, there's no artificial delay either. They are processed in the background after the capture, which includes encoding, analyzing and so on. Depending on other load it might take longer.

But for the issue as described, I do not see this being an issue - you can take lots of pictures in rapid succession - the capture itself has no actual delay and is NOT delayed by the processing of the pictures captured earlier. You can take pictures in rapid succession and then have them load in in a bit after they've been processed.

**Comment by code807** [comment]
@Frooxius I am consistently unable to take rapid photos. The time for the picture to load in is expected, I was referring to the taking itself. While using the gesture I will often take a photo and then immediately try to take another but the input will be ignored. I have to try it several times before it takes another. I can see the preview, so I know the camera is active, but I always have to wait a few seconds or spam the input before it will take another photo.

**Comment by Frooxius** [comment]
@code807 Having a video could help, but I think this is likely just due to how you capture them. There is no delay/timer before you can take another one, so it cannot be responsible for that.

I can take screenshots rapidly myself, so I know it is possible.

**Comment by code807** [comment]
After some testing, I believe my issue is actually with the way the game separates sequential photo gestures. I use an Index, and to take another finger photo I need to completely release the trigger and then completely depress it again. Instinctively, I would rather lessen my press on the trigger slightly to get away from a maximum press, then press it in again. It makes more sense to me as camera buttons are usually small buttons. The default camera object works the way I'd expect, and does not require the trigger to be fully released to take multiple pictures.

**Comment by Frooxius** [comment]
We could experiment with lowering the distance a bit, but it might result in a lot of accidental captures.

Unlike buttons, the gestures tend to be a lot more "fuzzy" and floaty, so they might lack the precision to do this well.
